programs/voter-stake-registry/src/events/mod.rs

```diff
@@ -37,3 +38,16 @@ pub struct DepositEntryInfo {
     /// Information about locking, if any
     pub locking: Option<LockingInfo>,
 }
+
+impl fmt::Debug for DepositEntryInfo {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("DepositEntryInfo")
+            .field("deposit_entry_index", &self.deposit_entry_index)
+            .field("voting_mint_config_index", &self.voting_mint_config_index)
+            .field("withdrawable", &self.withdrawable)
+            .field("voting_power", &self.voting_power)
+            .field("voting_power_deposit_only", &self.voting_power_deposit_only)
+            // .field("locking", &self.locking)
```

Comment:
could we somehow log this also? in a hurry I just ignored it

Reply:
fixed using `#[derive(Debug)]`